config.json

```diff
@@ -1507,51 +1824,79 @@
         "slug": "alphametics",
         "name": "Alphametics",
         "uuid": "7ed99829-acd8-4724-b92f-85146f1709f9",
-        "prerequisites": [],
-        "difficulty": 5,
-        "topics": [
-          "algorithms",
-          "backtracking",
-          "strings"
-        ]
+        "prerequisites": [
+          "strings",
+          "regular-expressions",
+          "multiple-clause-functions",
+          "pattern-matching",
+          "guards",
+          "case",
+          "cond",
+          "if",
+          "enum",
+          "charlists",
+          "ranges"
+        ],
+        "practices": ["strings", "enum", "pattern-matching"],
+        "difficulty": 7,
+        "topics": ["algorithms", "backtracking", "strings"]
       },
       {
         "slug": "pythagorean-triplet",
         "name": "Pythagorean Triplet",
         "uuid": "087a2a3b-71e2-4473-ab2f-3be5240567b5",
-        "prerequisites": [],
+        "prerequisites": ["integers", "ranges", "list-comprehensions"],
+        "practices": ["list-comprehensions"],
         "difficulty": 5,
-        "topics": [
-          "algorithms",
-          "math",
-          "reduce"
-        ]
+        "topics": ["algorithms", "math", "reduce"]
       },
       {
         "slug": "wordy",
         "name": "Wordy",
         "uuid": "087940a2-5372-4ff3-aa9f-ce2eb982d91c",
-        "prerequisites": [],
-        "difficulty": 6,
-        "topics": [
-          "parsers"
-        ]
+        "prerequisites": [
+          "strings",
+          "regular-expressions",
+          "multiple-clause-functions",
+          "pattern-matching",
+          "guards",
+          "case",
+          "cond",
+          "if",
+          "enum",
+          "errors",
+          "exceptions"
+        ],
+        "practices": ["regular-expressions", "pattern-matching"],
+        "difficulty": 7,
+        "topics": ["parsers"]
       },
       {
         "slug": "dot-dsl",
         "name": "Dot DSL",
         "uuid": "c316b8ef-7b2d-43d4-b402-f7d28b0cdaa7",
-        "prerequisites": [],
+        "prerequisites": [
+          "structs",
+          "regular-expressions",
+          "multiple-clause-functions",
+          "pattern-matching",
+          "guards",
+          "case",
+          "cond",
+          "if",
+          "enum",
+          "errors",
+          "exceptions",
+          "keyword-lists",
+          "nil",
+          "macros"
+        ],
+        "practices": ["keyword-lists", "structs", "macros"],
```

Comment:
It may be that I misunderstood the structure of this file -- but it seems to me that there is no such concept as `macros` (yet?).

Reply:
not yet, but there should be.  @angelikatyborska had a list of concepts that likely should be implemented at some point in one of her comments  ☝🏻 up there. Concepts that don't exist, _shouldn't_ affect things (afaik), but then this way we can be planning for their _hopeful_ addition someday

Reply:
We'll see what Jeremy or Erik say about this. I think I would personally prefer configlet to trip over non-existing concepts in prerequisites to warn me about typos. If we have to remove the non-existing concepts, they already have corresponding Github issues that list to which exercises it needs to be added.

Reply:
It is definitely good to list concepts that are not there yet, as the v3 website will just ignore them but you'll have them defined for when the concept is added later. configlet should indeed warn about using concepts in Practice Exercises that are not yet defined. For Concept Exercises, I feel like it should error if the concept is not specified.